src/main/java/me/totalfreedom/totalfreedommod/TotalFreedomMod.java

```diff
@@ -259,22 +259,31 @@ public void load(TotalFreedomMod plugin)
             try
             {
                 final Properties props;
+                final Properties gitprops;
                 try (InputStream in = plugin.getResource("build.properties"))
                 {
                     props = new Properties();
                     props.load(in);
                 }
+                try (InputStream in = plugin.getResource("git.properties"))
```

Comment:
Is there any way where we can include the git properties in the build properties? From a logical perspective, git properties signify the build process/state and it would make sense to centralize this data in one file.

Reply:
I did try to do this, but there was no easy way I could find (After 2 days of research) to do this without either not updating the git properties, or wiping the other build properties and wiping them back to their defaults. I'd like to have a better solution as this is certainly not perfect, but I couldn't come up with one unfortunately.